test/Mono.Linker.Tests/TestCasesRunner/LinkerArgumentBuilder.cs

```diff
@@ -184,6 +184,8 @@ public virtual void ProcessOptions (TestCaseLinkerOptions options)
 		{
 			if (options.CoreAssembliesAction != null)
 				AddCoreLink (options.CoreAssembliesAction);
+			else
+				AddCoreLink ("skip");
```

Comment:
This feels a bit off - how would we test the default behavior of the linker (That is omit the parameter)?
Not saying this it needs to be easy, but it feels wrong that we obviously don't test that behavior.

Reply:
I think such a test should be part of the integration like tests (basically different setup). This setup is primary about IL/metadata logic not about default behaviour.